Specs/DateTime/German/MergedExtractor.json

```diff
@@ -25,10 +25,10 @@
   {
     "Input": "Das ganze findet dann am Montag um 4 Uhr statt",
     "NotSupportedByDesign": "python",
-    "NotSupported":"javascript,dotnet",
+    "NotSupported": "javascript,dotnet",
```

Comment:
Will the fixes for these last cases come in a next PR?

Reply:
There is an issue with the Set-Extractor. Sometimes testcases are falsely detected as sets, which causes some tests to fail. I was not able to find the reason for this behaviour.